ground/src/main/java/com/google/android/ground/model/geometry/Geometry.kt

```diff
@@ -120,6 +138,10 @@ data class LinearRing(val coordinates: List<Coordinate>) : Geometry {
     this.maximum() >= other.maximum() && this.minimum() <= other.minimum()
 }
 
+/** Returns the center coordinate of the bounding box from the given list of coordinates. */
+private fun List<Coordinate>?.centerOrError(): Coordinate =
+  this?.map { Point(it) }?.toBounds()?.center() ?: error("missing vertices")
```

Comment:
I think this could work as a temporary fix, but using the center of the bounds will break for oddly shaped polygons. That said, implementation is non-trivial ([example](https://github.com/locationtech/jts/blob/master/modules/core/src/main/java/org/locationtech/jts/algorithm/Centroid.java)), so we may want to save that for future work.

Reply:
Added a note. Thanks!